### PR TITLE
allow extra config postgres with args

### LINKED ISSUE #538: change postgres settings
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
need to change postgres settings

##### ENVIRONMENT
* AWX version: 19.3
* Operator version: 0.13
* AWX install method: minikube

##### STEPS TO REPRODUCE

I need to change postgres setting to avoid production failures
https://github.com/ansible/awx/issues/10996#issuecomment-917686258

##### EXPECTED RESULTS

postgres should be run with modified settings

##### ACTUAL RESULTS

postgres is running with default settings

##### ADDITIONAL INFORMATION

https://github.com/ansible/awx/issues/10996#issuecomment-917686258
here is the problem and the way how to resolve it. Need to know how to implement this via awx-operator

### PR DESCRIPTION
**Pull request by chris93111**
allow change settings for postgresql with extra args in deployment

fix https://github.com/ansible/awx-operator/issues/538

```
postgres_extra_args:
  - '-c'
  - 'max_connections=1000'
```

### CODE REVIEW

File: roles/installer/templates/postgres.yaml.j2 [hunk 1]
```diff
@@ -41,6 +41,10 @@ spec:
         - image: '{{ postgres_image }}:{{ postgres_image_version }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           name: postgres
+{% if postgres_extra_args %}
+          command: postgres
```
Comment (shanemcd) [inline roles/installer/templates/postgres.yaml.j2]: I'm curious about this part. Does it not work without this line? The default entrypoint seems to do some stuff that we might not want to lose: https://github.com/docker-library/postgres/blob/ab940cbb923af99e2c7cf0e0ba5305bc6815aecc/12/bullseye/docker-entrypoint.sh

Comment (chris93111) [reply roles/installer/templates/postgres.yaml.j2]: yes it's work without, i delete

**Review by kdelee — APPROVED** [review]
This would be helpful

**Comment by rooftopcellist** [comment]
@chris93111 I have tested out your changes and they look good, but it is gated by CI.  Do you mind amending your commit, then re-pushing to this branch to kick of CI again?  Then we can get this merged.  Sorry for the hassle.

**Review by rooftopcellist — APPROVED** [review]

**Comment by chris93111** [comment]
@rooftopcellist no problem i do quickly

**Comment by rooftopcellist** [comment]
@chris93111 thank you for the fast response!  I actually just squashed your commits and opened this PR: https://github.com/ansible/awx-operator/pull/753, so no need to do the force-push if you haven't already.  We just need to wait for CI to run, then I'll get this merged.  Thank you for your contribution!

**Comment by rooftopcellist** [comment]
These changes merged as part of this PR: https://github.com/ansible/awx-operator/pull/753

**Pull request closed without merging**